app/p1-monitor/build.sbt

```diff
@@ -77,24 +77,24 @@ def staging(ov: Version, pv: Version) = AppConfig(
   distribution = List(Linux32, Linux64)
 ) extending List(common(ov, pv))
 
-// 2020A
-def v2020A(ov: Version, pv: Version) = AppConfig(
-  id = "2020A",
+// 2021A
```

Comment:
Often we keep two semester running due to de differences on the dates when they open/close

Reply:
So it should be 20B and 21A?

Reply:
Yes, though we'd only need 20B if we ever need to rebuild the previous p1 monitor, which has happened a few times